src/mainboard/amd/romecrb/src/main.rs

```diff
@@ -200,6 +203,110 @@ fn consdebug(w: &mut impl core::fmt::Write) -> () {
 }
 //global_asm!(include_str!("init.S"));
 
+// https://github.com/gz/rust-cpuid/issues/26#issuecomment-785280143
```

Comment:
These model functions probably needs to be in cpu::amd::something-or-other.

Reply:
Done.